evm-types.md

```diff
@@ -425,9 +425,16 @@ A cons-list is used for the EVM wordstack.
 
 -   Definedness conditions for `WS [ N ]` and `WS [ N := W ]`
 
-```{.k .symbolic}
-    rule #Ceil(WS[N])        => {((0 <=Int N) andBool (N <Int #sizeWordStack(WS))) #Equals true}  [anywhere]
-    rule #Ceil(WS[ N := W ]) => {((0 <=Int N) andBool (N <Int #sizeWordStack(WS))) #Equals true}  [anywhere]
+```{.k .symbolic-bytes}
+    rule #Ceil(WS[N])        => {((0 <=Int N) andBool (N <Int #sizeWordStack(WS)))            #Equals true}  [anywhere]
+    rule #Ceil(WS[ N := W ]) => {((0 <=Int N) andBool (N <Int #sizeWordStack(WS)))            #Equals true}  [anywhere]
```

Comment:
Can't we just make `_[_]` and `_[_:=_]` total instead of adding definedness conditions? I think it should be possible pretty simply by adding cases:

```
rule .WordStack [ 0 ] => 0

rule .WordStack [ N := WS ] => #take(N, .WordStack) ++ WS
```

Then we don't need to maintain definedness conditions.